src/main/java/de/hbz/lobid/helper/EtikettMaker.java

```diff
@@ -166,6 +166,7 @@ private void initMaps(InputStream labelIn) {
 		try (InputStream in = labelIn) {
 			return new ObjectMapper().readValue(in, type);
 		} catch (Exception e) {
+			e.printStackTrace();
```

Comment:
I think the bug is at https://github.com/hbz/lobid-rdf-to-json/blob/master/src/main/java/de/hbz/lobid/helper/EtikettMaker.java#L121 .  Exception during initialization. At least must be logged as a warning - maybe it would even be better to exit the process by throwing an Error?